source/run-command/tests/unified/run-command.yml

```diff
@@ -0,0 +1,242 @@
+description: runCommand spec requirements
+
+schemaVersion: "1.5"
+
+createEntities:
+  - client:
+      id: &client client
+      observeEvents: [commandStartedEvent]
+  - database:
+      id: &plainDb plainDb
+      client: *client
+      databaseName: *plainDb
+  - collection:
+      id: &collection collection
+      database: *plainDb
+      collectionName: *collection
+  - database:
+      id: &dbWithRC dbWithRC
+      client: *client
+      databaseName: *dbWithRC
+      databaseOptions:
+        readConcern:
+          level: 'local'
+  - database:
+      id: &dbWithWC dbWithWC
+      client: *client
+      databaseName: *dbWithWC
+      databaseOptions:
+        writeConcern:
+          w: 0
+  - session:
+      id: &session session
+      client: *client
+  # Stable API test
+  - client:
+      id: &stableApiClient stableApiClient
+      observeEvents: [commandStartedEvent]
+      serverApi:
+        version: "1"
+        strict: true
+  - database:
+      id: &stableApiDb stableApiDb
+      client: *stableApiClient
+      databaseName: *stableApiDb
+
+initialData:
+- collectionName: *collection
+  databaseName: *plainDb
+  documents: [ { _id: 0 } ]
```

Comment:
Do any of the tests actually need this document to exist?

If you only need the collection to exist for the transaction test you can use `documents: []` here.